client/hotrod-client/src/test/java/org/infinispan/client/hotrod/ConsistentHashV2IntegrationTest.java

```diff
@@ -147,7 +146,7 @@ public void testCorrectBalancingOfKeysAfterNodeKill() {
          log.tracef("Client topology id is %d, waiting for it to become %d", topologyId,
                topologyIdBeforeJoin + 2);
          // The put operation will update the client topology (if necessary)
```

Comment:
comment should be removed I think

Reply:
@karesti I replaced `put` with `get`, sorry for the oversight.